rust/src/push/mod.rs

```diff
@@ -328,10 +328,16 @@ pub enum Condition {
 #[serde(tag = "kind")]
 pub enum KnownCondition {
     EventMatch(EventMatchCondition),
+    // Identical to event_match but gives predefined patterns. Cannot be added by users.
+    #[serde(skip_deserializing, rename = "event_match")]
+    EventMatchType(EventMatchTypeCondition),
```

Comment:
Can we add a test that this serializes correctly, for paranoia.

Reply:
Done! I added deserialization ones too.